manifests/supervisorcluster/1.20/cns-csi.yaml

```diff
@@ -423,3 +424,202 @@ spec:
   selector:
     app: vsphere-csi-controller
   type: LoadBalancer
+---
```

Comment:
extra line

Reply:
reverted tkgs-ha-related changes for the 1.20 release.